sklearn/feature_extraction/image.py

```diff
@@ -262,6 +262,7 @@ def _compute_n_patches(i_h, i_w, p_h, p_w, max_patches=None):
         The maximum number of patches to extract. If max_patches is a float
         between 0 and 1, it is taken to be a proportion of the total number
         of patches.
+        If max_patches=None then the total number of patches will be extracted.
```

Comment:
To be consistent with the public function documentation already done in `main`, I propose the following change.

```suggestion
        of patches. If `max_patches` is None it corresponds to the total number
        of patches that can be extracted.
```